less/about-page.less

```diff
@@ -0,0 +1,408 @@
+                margin: 0%;
+            }
+            h3{
+                font-size: 2.5rem;
+                padding-bottom: 5%;
+                color: @chalk;
+            }
+            img{
+                height: 100px;
+                padding-bottom: 5%;
+                color: @chalk;
+            }
+            p{
+                padding-bottom: 8%;
+                font-size: 1.4rem;
+                color: @chalk;
+            }
+        }
+    }
+    .thirdRow{
+        display: flex;
+        justify-content: space-between;
+        padding-bottom: 10%;
+        @media @phone{
+            flex-direction: column;
+            align-items: center;
+        }
+        div{
+            width: 300px;
+            background-color: @lightGreen;
+        }
+        .sandra{
+            .leftPerson();
+            border-radius: 10px;
+            @media @phone{
+                margin-left: 0%;
+                margin-bottom: 10%;
+            }
+            h3{
+                font-size: 2.5rem;
+                padding-bottom: 5%;
+                color: @chalk;
+            }
+            img{
+                height: 100px;
+                padding-bottom: 5%;
+            }
+            p{
+                padding-bottom: 8%;
+                font-size: 1.4rem;
+                color: @chalk;
+            }
+        }
+        .caitlyn{
+            .rightPerson();
+            border-radius: 10px;
+            @media @phone{
+                margin: 0%;
+            }
+            h3{
+                font-size: 2.5rem;
+                padding-bottom: 5%;
+                color: @chalk;
+            }
+            img{
+                height: 100px;
+                padding-bottom: 5%;
+            }
+            p{
+                padding-bottom: 8%;
+                font-size: 1.4rem;
+                color: @chalk;
+            }
+
+        }
+    }
+    .fourthRow{
+        display: flex;
+        justify-content: space-between;
+        padding-bottom: 10%;
+        @media @phone{
+            flex-direction: column;
+            align-items: center;
+        }
+        div{
+            width: 300px;
+            background-color: @lightGreen;
+        }
+        .jennie{
+            .leftPerson();
+            border-radius: 10px;
+            @media @phone{
+                margin-left: 0%;
+                margin-bottom: 10%;
+            }
+            h3{
+                font-size: 2.5rem;
+                padding-bottom: 5%;
+                color: @chalk;
+            }
+            img{
+                height: 100px;
+                width: 100px;
+                padding-bottom: 5%;
+            }
+            p{
+                padding-bottom: 8%;
+                font-size: 1.4rem;
+                color: @chalk;
+            }
+        }
+    }
+   
+}
+.topPhotos{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background: @darkGreen;
+    @media @iphone{
```

Comment:
is this correct? `@media @iphone` or was it meant to be `@media @phone`?

Reply:
Thank you Izzy.  It was intentional, the photos looked terrible on my phone, but I wanted to keep them on my page to be graded.